src/Pub/API/DTOs/ProjectDto.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace API.DTOs
+{
+    public class ProjectDto
```

Comment:
Ensure this aligns with the ERD.

<img width="191" alt="Screen Shot 2019-09-16 at 1 38 59 AM" src="https://user-images.githubusercontent.com/4195689/64938281-f86ea800-d822-11e9-9790-7d24b5655228.png">